### PR TITLE
Merge release 1.69.1 to develop

### PR DESCRIPTION
**Pull request by mkevins**
This PR merges the 1.69.1 betafix release back to `develop`.

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes [more info](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/Release-notes.md) and have added them to [RELEASE-NOTES.txt](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) if necessary.

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -7,16 +7,21 @@ Unreleased
 * [*] Fix missing translations of color settings [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4479]
 * [*] Fix cut-off setting labels by properly wrapping the text [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4475]
 * [*] Highlight text: fix applying formatting for non-selected text [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4471]
+* [**] Fix Android handling of Hebrew and Indonesian translations [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4397]
```
Comment (ttahmouch) [inline RELEASE-NOTES.txt:10]: I see this was moved to `Unreleased` as it sounds like it was merged to `develop` *after* we cut the beta fix. I haven't confirmed timing of anything, but I saw that there was a pending question [here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4397#issuecomment-1016954541). Was this confirmed?

**Review by ttahmouch — APPROVED** [review]
The changes LGTM. Thanks for updating `develop`, @mkevins . :)

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (mkevins) [reply RELEASE-NOTES.txt:10]: Yeah, this looks like it was auto-resolved incorrectly by git (the PR had been created prior to the release branch, but it didn't land until after the cut, so it was actually in the "Unreleased" section of the notes in the PR itself. It seems that [when it was merged](https://github.com/wordpress-mobile/gutenberg-mobile/commit/f376fdb685091ad168e1af3e3216ae4a33af0923) into `develop`, the new version headings were already present in the notes, but this line was added under them, when it should have been "bumped" back up to the "new" "unreleased" section.

The pending question there would be a good confirmation, but is mostly relevant for the current release wranglers of the `1.70.0` release, so they have clarity on where that note belongs.

**Comment by mkevins** [comment]
Thanks for reviewing @ttahmouch !

**Pull request merged**